common/test/acceptance/pages/studio/index.py

```diff
@@ -269,13 +271,61 @@ def language_selector(self):
         )
         return self.q(css='#settings-language-value')
 
+    def get_nav_help_css(self):
+        """
+        Returns the unique css of nav help DOM element
+        """
+        return NAV_HELP_CSS
+
+    def click_nav_help(self):
+        """
+        Clicks on the help link in the navigation bar.
+        """
+        click_css(page=self, css=NAV_HELP_CSS, require_notification=False)
+
+    def get_side_bar_help_css(self):
+        """
+        Returns the unique css of side bar help DOM element
+        """
+        return SIDE_BAR_HELP_AS_LIST_ITEM
+
+    def click_side_bar_help(self):
+        """
+        Clicks on the help link in the side bar.
+        """
+        click_css(page=self, css=SIDE_BAR_HELP_AS_LIST_ITEM, require_notification=False)
+
 
 class HomePage(DashboardPage):
     """
     Home page for Studio when logged in.
     """
     url = BASE_URL + "/home/"
 
+    def get_nav_help_css(self):
+        """
+        Returns the unique css of nav help DOM element
+        """
+        return NAV_HELP_CSS
+
+    def click_nav_help(self):
+        """
+        Clicks on the help link in the navigation bar.
+        """
+        click_css(page=self, css=NAV_HELP_CSS, require_notification=False)
+
+    def get_side_bar_help_css(self):
+        """
+        Returns the unique css of side bar help DOM element
+        """
+        return SIDE_BAR_HELP_AS_LIST_ITEM
+
+    def click_side_bar_help(self):
+        """
+        Clicks on the help link in the side bar.
+        """
+        click_css(page=self, css=SIDE_BAR_HELP_AS_LIST_ITEM, require_notification=False)
+
```

Comment:
@Muddasser Again. Why we have same code(methods) in two different classes in this file?